README.md

```diff
@@ -1,50 +1,90 @@
+---
 
-For Android users, download the APK file from [Snapshot Build](https://downloads.cardforge.org/dailysnapshots/) to your device. 
-On first run, Forge will download all needed data.
+## 🛠️ Installation Guide
 
-## Modes of Play
+### 📥 Desktop Installation
+1. **Latest Releases:** Download the latest version [here](https://github.com/Card-Forge/forge/releases/latest).
+2. **Snapshot Build:** For the latest development version, grab the `forge-gui-desktop` tarball from our [Snapshot Build](https://downloads.cardforge.org/dailysnapshots/).
+   - **Tip:** Extract to a new folder to prevent data loss.
+3. **User Data Management:** Previous players’ data is preserved during upgrades.
+4. **Java Requirement:** Ensure you have **Java 8 or later** installed.
 
-Forge has a variety of ways to play the game. The most popular way is our Adventure mode, which is a single player campaign that allows you to play against a variety of AI opponents. 
-You walk around an overworld map, and can challenge opponents to games of Magic. As you play, you'll collect more cards and items to improve your abilities.
+### 📱 Android Installation
+- Download the **APK** from the [Snapshot Build](https://downloads.cardforge.org/dailysnapshots/). On the first launch, Forge will automatically download all necessary assets.
 
-Check the [Gameplay Guide](https://github.com/Card-Forge/forge/wiki/Gameplay-Guide) for more info.
+---
+
+## 🎮 Modes of Play
+
+Forge offers various exciting gameplay options:
+
+### 🌍 Adventure Mode
+Embark on a thrilling single-player journey where you can:
+- Explore an overworld map.
+- Challenge diverse AI opponents.
+- Collect cards and items to boost your abilities.
 
 ![Adventure Mode](https://downloads.cardforge.org/images/site/adventure-mode.png "Adventure Mode")
 
+### 🔍 Quest Modes
+Engage in focused gameplay without the overworld exploration—perfect for quick sessions!
+
+### 🤖 AI Formats
+Test your skills against AI in multiple formats:
+- **Sealed**
+- **Draft**
+- **Commander**
+- **Cube**
+
+For comprehensive gameplay instructions, visit our [Gameplay Guide](https://github.com/Card-Forge/forge/wiki/Gameplay-Guide).
+
+---
+
+## 💬 Support & Community
+
+Need help? Join our vibrant Discord community! 
+- 📜 Read the **#rules** and explore the **FAQ**.
+- ❓ Ask your questions in the **#help** channel for assistance.
+
+---
+
+## 🤝 Contributing to Forge
+
+We love community contributions! Interested in helping? Check out our [Contributing Guidelines](CONTRIBUTING.md) for details on how to get started.
+
+---
+
+## ℹ️ About Forge
 
-Forge has several Quest modes, which is similar but without the overworld map. 
+Forge aims to deliver an immersive and customizable Magic: The Gathering experience for fans around the world. 
 
-You can also play against the AI in a variety of formats, such as Sealed, Draft, Commander and Cube.
+### 📊 Repository Statistics
+- **⭐ Stars:** 969
```

Comment:
Is this supposed to be

```
# My Project

![GitHub stars](https://img.shields.io/github/stars/your-username/your-repo?style=social)
![GitHub forks](https://img.shields.io/github/forks/your-username/your-repo?style=social)
![GitHub contributors](https://img.shields.io/github/contributors/your-username/your-repo)

```

Reply:
I added the repository statistics section for reference. If you think it's unnecessary, I'm happy to remove it

Reply:
What I'm saying is you added static values, I believe what I listed above would give you shields that always list the exact number.

Reply:
Can you go through the edits I made and check if there are any changes to be made?